orchestration/templates/ingest-hca.yaml

```diff
@@ -694,3 +698,22 @@ spec:
         command: [bash]
         source: |
         {{- include "argo.render-lines" (.Files.Lines "scripts/get-outdated-row-ids.sh") | indent 10 }}
+
+    ##
+    ## Move any logs to GCS.
+    ##
+    - name: move-error-logs-to-gcs
+      inputs:
+        parameters:
+          - name: table
```

Comment:
why are we passing in table here? I don't see it being used, or am I missing it?

Reply:
Good catch! Thank you! This actually doesn't need any parameters so I just removed it.